samples/sampler/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 The following is a curated list of changes in the Enact Sampler, newest changes on the top.
 
+## unreleased
+
+- Updated `babel-loader` version to `^9.1.2`.
```

Comment:
Interesting. I did not notice this changelog file for the moonstone sampler. It seems it hasn't been updated for 4 years. @seunghoh , do we stil need this file?

Reply:
Please revert samples/sampler/CHANGELOG.md
(This change does not need to be written)

Reply:
@daniel-stoian-lgp hm.. let me think about it. Anyway, I don't think this change log doesn't need. We actually don't put the dependency change in the change log